plugins/main/public/components/common/welcome/agents-welcome.js

```diff
@@ -326,14 +326,17 @@ export const AgentsWelcome = compose(
           </EuiFlexItem>
           <EuiFlexItem grow={false} className='wz-module-header-agent-title'>
             <EuiFlexGroup responsive={false} gutterSize='none'>
-              <EuiFlexItem grow={false} style={{ marginTop: 7 }}>
+              <EuiFlexItem
+                grow={false}
+                style={{ marginTop: 7, marginRight: '0.5rem' }}
+              >
                 <ButtonExploreAgent />
               </EuiFlexItem>
               <EuiFlexItem grow={false} style={{ marginTop: 7 }}>
                 <WzButton
                   buttonType='empty'
                   iconType='inspect'
-                  onClick={() => this.props.switchTab('syscollector')}
+                  onClick={() => this.props.switchTab(AgentTabs.SOFTWARE)}
```

Comment:
**question**: why the value of the tab was renamed from `syscollector` to `software`?

Reply:
Since the `syscollector` tab no longer exists, I have selected the first tab to be displayed when you click on `Inventory Data`.